dialogue-core/src/main/java/com/palantir/dialogue/core/TimingEndpointChannel.java

```diff
@@ -68,24 +79,39 @@ static EndpointChannel create(Config cf, EndpointChannel delegate, Endpoint endp
 
         return DialogueFutures.addDirectCallback(response, new FutureCallback<Response>() {
             @Override
-            public void onSuccess(Response _result) {
-                updateResponseTimer();
+            public void onSuccess(Response response) {
```

Comment:
The code here mostly simplifies/replicates the retry decisions in RetryingChannel; I think it's ok to start this way, but once we determine that these metrics are the right ones, we could either move this timing down to that channel, or possibly have a field on the response that RetryingChannel sets?